.setup/submitty_sudoers

```diff
@@ -0,0 +1,2 @@
+submitty_daemon ALL = (root) NOPASSWD: /usr/local/submitty/sbin/create_course.sh
+submitty_daemon ALL = (root) NOPASSWD: /usr/local/submitty/sbin/adduser.py
```

Comment:
Discussed this with Barb and we're good to go with using this file. I'm fine leaving this as is for now and will replace the hardcoded stuff with variables (in this and the crontab file) in a future PR as part of the larger issue of #4011.